modules/hal_nxp/mcux/mcux-sdk-ng/device/device.cmake

```diff
@@ -38,7 +38,9 @@ endif()
 
 set(CONFIG_MCUX_COMPONENT_device.system ON)
 set(CONFIG_MCUX_COMPONENT_device.CMSIS ON)
-set(CONFIG_MCUX_COMPONENT_driver.clock ON)
+if (NOT CONFIG_CLOCK_CONTROL_ARM_SCMI)
```

Comment:
no space in cmake between if and bracket

Reply:
fixed, thanks.